src/useChildrenAreWrapped/useChildrenAreWrapped.ts

```diff
@@ -0,0 +1,30 @@
+import { computed, ComputedRef, ref, Ref } from 'vue'
+import { useElementRect } from '@/useElementRect/useElementRect'
+import { getComputedStyleValue } from '@/utilities/getComputedStyleValue'
+import { getNumberFromPx } from '@/utilities/getNumberFromPx'
+
+export function useChildrenAreWrapped(children: HTMLElement[] | Ref<HTMLElement[]>, container: HTMLElement | Ref<HTMLElement | undefined>): ComputedRef<boolean> {
+  const childrenRef = ref(children)
+  const containerRef = ref(container)
+
+  const { width } = useElementRect(containerRef)
+
+  function getContainerGap(): number {
+    if (!containerRef.value) {
+      return 0
+    }
+
+    const gapInPx = getComputedStyleValue(containerRef.value, 'gap')
+
+    return getNumberFromPx(gapInPx)
+  }
+
+  const childrenWidthSum = computed(() => {
+    const rects = childrenRef.value.map(useElementRect)
+    const gap = getContainerGap()
+
+    return rects.reduce((sum, { width }) => sum += width.value + gap, 0)
+  })
```

Comment:
I'm not sure this technique is solid. Elements can have margins that would cause them to wrap sooner than this calculation would determine. Also depending on the box model borders could also effect when the elements would wrap.

Reply:
might not work in 100% of circumstances but it definitely solved my problem 🤷

Reply:
Might be worth noting its limitations then if we know there are limitations

Reply:
I think I've accounted for such limitations, and updated readme. Take a look at let me know what you think